include/ck/utility/f8_utils.hpp

```diff
@@ -44,7 +60,7 @@ __host__ __device__ Y run_cast_to_f8(X x, uint32_t rng)
 
     // convert to bitwise
     using T_bitwise     = typename NumericUtils<X>::bitwise_type;
-    T_bitwise x_bitwise = *(reinterpret_cast<T_bitwise*>(&x));
+    T_bitwise x_bitwise = bitcast<T_bitwise, X>(x);
```

Comment:
Here and bellow, can you use bit_cast from include/ck/utility/type.hpp:41

Reply:
I haven't seen it. Thanks for the suggestion!

Since `bit_cast` is `constexpr` I was able to make `Inf`, `Nan`, ... `constexpr` too.

Reply:
Thanks